packages/create-cloudflare/src/dialog.ts

```diff
@@ -27,18 +27,27 @@ export function createDialog(lines: string[]) {
 export function printWelcomeMessage(
 	version: string,
 	telemetryEnabled: boolean,
+	args: Partial<C3Args>,
 ) {
 	const lines = [
 		`👋 Welcome to create-cloudflare v${version}!`,
 		`🧡 Let's get started.`,
 	];
 
+	if (args.experimental) {
+		lines.push("", blue`🧪 Running in experimental mode`);
+	}
+
 	if (telemetryEnabled) {
+		if (args.experimental) {
```

Comment:
this seems unnecessary 🤔  ... why is it needed?

Reply:
it's just so that if you are in experimental mode and telemetry is enabled, there is a clean clean newline between the two, as you can see in the test snapshot:
![Screenshot 2024-12-10 at 15 30 58](https://github.com/user-attachments/assets/153ac9a5-e106-42d4-9aeb-cd05be6b259c)

without this the telemetry first line would be right below the external one, which, given the fact that the telemetry into already has an empty line itself, would make things pretty unclear in my opinion

Reply:
oh, gotcha! thank you for pointing that out 🙏

Reply:
one last question....in that case, wouldn't it make more sense to 👇 ? 
```
if (args.experimental) {
		lines.push("", blue`🧪 Running in experimental mode\n`);
	}
```

Reply:
right right...no let's just leave it as is. it'll be alright